### ISSUE TITLE
Infura/Etherscan request optimization

### ISSUE THREAD

**Issue by rachelhamlin** [status-im/status-react#9471]
# Problem

Placeholder issue: Ensure we have an adequate # of balance requests from Infura and/or Etherscan before v1. 

Related issue - #8746

## Implementation

Known steps towards feature implementation.
What needs further specifying and investigating.

## Acceptance Criteria

Rules for the future PR to be accepted.

## Notes

Random notes to keep in mind while implementing it. Mostly about related issues and future plans and thoughts.

## Future Steps

Steps which should be taken after this issue has been resolved.

**Comment by rachelhamlin** [comment]
We're seeing a fairly regular pattern of failure by Infura to provide data in a timely fashion.

The ENS setting flow threw me [three errors/bugs](https://github.com/status-im/status-react/pull/9437#issuecomment-555915861) as a result of Infura failures.

**Comment by rachelhamlin** [comment]
I specifically seem to have some grave Infura issue on my device. When I tested the image sending PR, I got these errors:

<img src="https://user-images.githubusercontent.com/15470435/70062322-b6613100-15dd-11ea-8301-2ac16a1c3f63.jpeg" width=250/>
<img src="https://user-images.githubusercontent.com/15470435/70062299-ae08f600-15dd-11ea-852b-68a0c6f1f9ea.jpeg" width=250/>
<img src="https://user-images.githubusercontent.com/15470435/70062314-b2351380-15dd-11ea-875d-2bad108fc13a.jpeg" width = 250/>

**Comment by rachelhamlin** [comment]
Notes from call with Infura in December: https://notes.status.im/785IoPWcQvuUpl7JygLEog

Older notes on potential alternatives: https://notes.status.im/xLZkbUztQQeDUPX_I7qRKg

**Comment by rachelhamlin** [comment]
@flexsurfer @andremedeiros just to track what's happened here...

We upgraded our Infura subscription to a Growth plan for full archival data. 
That unblocked Roman's work on [Tx history](https://github.com/status-im/status-react/pull/9744). 

We did _not_ make any other changes to use Cloudflare in place of Infura afaik. Correct? Maybe that satisfies this issue for v1. Future optimizations possible.
